.github/ISSUE_TEMPLATE/4_decommission-hub.md

```diff
@@ -38,6 +38,7 @@ Usually, it is because it was a hub that we created for a workshop/conference an
   - `helm --namespace HUB_NAME delete HUB_NAME`
   - `kubectl delete namespace HUB_NAME`
 - [ ] Delete the hub's authentication application on auth0, GitHub or CILogon
+- [ ] Remove the [uptime check](https://console.cloud.google.com/monitoring/uptime?referrer=search&project=two-eye-two-see) and associated alert policy for this hub
```

Comment:
I think we should also have instructions about the manual steps in the corresponding  "add hub" (or rather "add cluster") docs section?

*I think* but I'm not sure that the only manual step is to take per clsuter is to create the notification channel ?

Reply:
I've tried to answer this in https://github.com/2i2c-org/infrastructure/pull/1750/files#diff-f4ac06565f573ca00491b1116529d5b76e373e63f87defa64662235f8e79e739